### ISSUE TITLE
Storybook v.6 controls error

### ISSUE THREAD

**Issue by lawlesscreation** [titonobre/storybook-addon-angularjs#23]
Hi, I've been testing this out on a legacy web application build in AngularJS and so far it's been great. I've hit a snag though in that I get the following error when using it with Storybook v.6 (I believe some changes happened with knobs > Controls in the latest version".

When changing a control input I get:
```
TypeError: Cannot read property 'invoke' of undefined
```

My story looks like this (as per the storybook examples of how to use controls):
```
export const simpleAlert = (args) => ({
  template: `
  <my-alert data-title="${args.title}">
    ${args.content}
  </my-alert>
`});
simpleAlert.args = {
  content: 'content',
  title: 'hello'
};
```

Changing either of the content or title control causes this error, I can see the demo examples are all using Storybook v.5. I wondered if you'd had any experience using the new v.6? I'm going to downgrade to v.5 and see if I have any more success.

Thanks for the great work.

**Comment by lawlesscreation** [comment]
As additional notes. It seems to work fine when I use `@storybook/addon-knobs` but not when using the `controls` addon.

**Comment by titonobre** [comment]
Hi @lawlesscreation 

I've been trying for the past few weeks to update the examples to use Storybook 6 but I'm lacking some free time to do it. However I don't expect big changes. I use this addon at work in a big project and there is already a PR open by a colleague and changes were essentially what's on the migration guide.

Now, regarding the new Controls feature.

This addon depends on the Storybook HTML Framework, which [does not support Controls yet](https://storybook.js.org/docs/html/api/frameworks-feature-support). I haven’t digged enough into this topic, so I don't know how it works internally. 

If you happen to find more information on this, please let me know.

**Comment by titonobre** [comment]
Hi @lawlesscreation 

I've updated the examples and the addon to work with Storybook 6. I also updated the decorator to support the new story args, which are used by the controls addon.

Commit here: https://github.com/titonobre/storybook-addon-angularjs/commit/bda2bd4e367b459372a5cb2d9a6c7a77de2f9948

Story here: https://storybook-addon-angularjs.now.sh/?path=/story/demos-csf-demos-2--with-args

This was released in [storybook-addon-angularjs@1.4.0](https://www.npmjs.com/package/storybook-addon-angularjs/v/1.4.0)

**Comment by lawlesscreation** [comment]
Thanks @titonobre I'll give it a go!

I've had some great success using the addon so far, thanks for your amazing work.
